main.js

```diff
@@ -13,4 +15,17 @@ function rotate(e) {
   }
 }
 
+function moveCar(e) {
+  if (movePixel > 1000) return;
+  movePixel += 5;
+  $carContainer.style.transform = `translate(${movePixel}px)`;
```

Comment:
I would recommend doing this with `absolute` positioning and changing the value of the `top` and `left` properties. It will make movement after turning easier to implement 👍